src/umpire/strategy/MonotonicAllocationStrategy.cpp

```diff
@@ -51,9 +51,7 @@ MonotonicAllocationStrategy::allocate(size_t bytes)
 
 void
 MonotonicAllocationStrategy::deallocate(void* UMPIRE_UNUSED_ARG(ptr))
-{
-  UMPIRE_LOG(Info, "() doesn't do anything");
```

Comment:
Why was this removed?  Was it causing a problem?

Reply:
Good question. @corbett5 ?

Reply:
This was removed because although `MonotonicAllocationStrategy::deallocate` alone doesn't do anything when wrapped in `AllocationTracker` the `deallocate` method does have side effects. My opinion is that a message like this will discourage the user from calling `deallocate`.

Reply:
It doesn't bother me since this was not implemented by me.  But it is a change of behavior and I wanted to make sure @davidbeckingsale saw it.  Since he has already approved this PR, I will drop it.